sentry/src/test/java/io/sentry/vendor/gson/internal/bind/util/ISO8601UtilsTest.java

```diff
@@ -1,9 +1,22 @@
-package io.sentry.vendor.gson.internal.bind.util;
+/*
+ * Adapted from https://github.com/FasterXML/jackson-databind/blob/c1e92435c6942386394a2a7733065bb047773107/src/test/java/com/fasterxml/jackson/databind/util/ISO8601UtilsTest.java
+ *
+ * Copyright (C) 2007-, Tatu Saloranta
```

Comment:
It seems this is wrong; `ISO8601UtilsTest.java` was retroactively added to Gson by https://github.com/google/gson/pull/1638 and is not based on jackson-databind's `ISO8601UtilsTest.java`.

(Please correct me if I am wrong though)